internal/slicer/slicer_test.go

```diff
@@ -616,6 +616,19 @@ var slicerTests = []slicerTest{{
 }, {
 	summary: "Can list parent directories of normal paths",
 	slices:  []setup.SliceKey{{"test-package", "myslice"}},
+	pkgs: map[string][]byte{
+		"test-package": testutil.MustMakeDeb(append(
+			testutil.TestPackageEntries,
+			// This particular path starting with "/check" is chosen to test for
```

Comment:
I think relying on the fact that `c.MkDir` creates a directory named `check-` is not something that we want to do, if the library changes that internal implementation detail then the test is not meaningful.

Maybe it is feasible to do it by changing `o.TargetDir` using the `test.hackopt` field. You can then add characters there.

Reply:
Alright, yeah that would be better. I will update it soon.

Reply:
Updated in https://github.com/canonical/chisel/pull/145/commits/5fe11069c55975da90c79d69b1dd705389302e49.